oggm/utils/_funcs.py

```diff
@@ -780,3 +781,96 @@ def shape_factor_adhikari(widths, heights, is_rectangular):
     shape_factors[np.isnan(shape_factors)] = 1.
 
     return shape_factors
+
+
+def cook_rgidf(gi_gdf, region='13', version='60', ids=None,
+               id_suffix=None, save_special_columns=None,
+               assign_col_values=None):
+    """ Cook the user's glacier inventory with the RGI format
+
+    Parameters
+    ----------
+    gi_gdf : :py:geopandas.GeoDataFrame
+        the GeoDataFrame of the user's glacier inventory
+    region : str
+        Glacier RGI region code, only usable when set `confirm_region=False`.
+        The default is '13'.
```

Comment:
There should be no default here - it should fail if the region is not provided and cannot be inferred automatically